src/com/amazon/corretto/crypto/provider/AmazonCorrettoCryptoProvider.java

```diff
@@ -231,6 +231,7 @@ private void addSignatures() {
 
     if (shouldRegisterMLDSA) {
       addService("Signature", "ML-DSA", "EvpSignatureRaw$MLDSA");
+      addService("Signature", "ML-DSA-Ext-Mu", "EvpSignatureRaw$MLDSAExtMu");
```

Comment:
There's currently no JCA-standard name for this (yet?), so we have some latitude in choosing the algorithm name. I'm not sold on `ML-DSA-Ext-Mu`, open to suggestions from reviewers.

Reply:
In [draft-ietf-lamps-dilithium-certificates](https://datatracker.ietf.org/doc/draft-ietf-lamps-dilithium-certificates/) we use `ExternalMu-ML-DSA`. In [ACVP ML-DSA JSON Specification](https://pages.nist.gov/ACVP/draft-celi-acvp-ml-dsa.html) it is just referred to as `externalmu`.

Notably, theres no hyphen between `external` and `mu` in both.

We chose `ExternalMu-ML-DSA` in the IETF to hint at consistency with NISTs notion of prefixing the mode name distinction: i.e.,`HashML-DSA`.

As such `ML-DSA-ExtMu`, `ML-DSA-ExternalMu` are good options. If you like  `ML-DSA-Ext-Mu`, `ML-DSA-External-Mu` with the extra hyphen, that's also fine!